cc-pk-service/src/handleData.ts

```diff
@@ -0,0 +1,200 @@
+
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { Survey, ItemInfo, Item, Building } from './types';
+
+dotenv.config();
+
+// Variables
+const username: string = process.env.PK_USERNAME ?? '';
+const password: string = process.env.PK_PASSWORD ?? '';
+const c_id: string = process.env.PK_CLIENTID ?? '';
+const g_type: string = process.env.PK_GRANTTYPE ?? '';
+
+let userToken:string;
+export let userId:string;
+
+// Get usertoken and userid
+getUserInfo();
+
+/**
+ * Initalize an empty ItemInfo -object
+ * @returns An empty ItemInfo -object
+ */
+function initInfo() {
+    const info:ItemInfo = {
+        id:'',
+        componentName:'',
+        reusableMaterialId:'',
+        usability:'',
+        amount:0,
+        unit:'',
+        description:'',
+        images: '',
+        amountAsWaste:0,
+        metadata:{
+            creatorId:'',
+            createdAt:'',
+            lastModifierId:'',
+            modifiedAt: ''
+        }
+    }
+    return info;
+}
+
+/**
+ * Collect user info at the beginning
+ */
+async function getUserInfo() {

```

Comment:
this function could be in another file (utils.ts/helpers.ts), since it's not used in the data functions?

Reply:
True, but it would be kind of difficult since it uses getToken and getId -functions and saves information about userToken and userId. Since the end goal is that PK service receives userId and userToken from the frontend (if I understood correctly), this function and also getToken and getId -functions can be deleted entirely once PK service functions in a way that it can receive parameters/headers alongside the requests that it receives. So yes, I could start moving these to another file but I'm not sure if it's necessary since they will be deleted at some point.
 
But if even after this comment you still think I should move this function somewhere else, then I'll do that.